pkg/util/cluster/cluster.go

```diff
@@ -60,7 +61,7 @@ type Cluster struct {
 	openshiftclustersv20200430        redhatopenshift20200430.OpenShiftClustersClient
 	openshiftclustersv20210901preview redhatopenshift20210901preview.OpenShiftClustersClient
 	openshiftclustersv20220401        redhatopenshift20220401.OpenShiftClustersClient
-	openshiftclustersv20220904        redhatopenshift20231122.OpenShiftClustersClient
+	openshiftclustersv20230904        redhatopenshift20230904.OpenShiftClustersClient
 	openshiftclustersv20231122        redhatopenshift20231122.OpenShiftClustersClient
```

Comment:
I don't see why simply instantiating this client without using it would break anything, but we could consider removing it completely to stay on the safe side.

Reply:
The specific failures we saw during our RP deployment were when we attempted to call CreateOrUpdate on the 20231122 client, not during instantiation, so I think this is safe. I'll remove it anyways though. 

My overarching concern here is that I don't think we need more than one client here at all, but I'll tackle that in a more involved refactor that we don't hotfix.